chapters/equations.tex

```diff
@@ -912,8 +912,9 @@ \subsection{Equations Needed for Initialization}\label{the-number-of-equations-n
 \end{lstlisting}
 
 Here we have three variables with unknown values: two dynamic variables that also are state variables, \lstinline!x1! and \lstinline!x2!, i.e., $n=2$, one output variable \lstinline!y!, i.e., $m=1$, and one input variable \lstinline!u! with known value.
-A consistent solution of the initial value problem providing initial values for \lstinline!x1!, \lstinline!x2!, \lstinline!der(x1)!, \lstinline!der(x2)!, and \lstinline!y! needs to be found.
-Two additional initial equations thus need to be provided to solve the initialization problem.
+A consistent solution of the initialization problem requires finding initial values for \lstinline!x1!, \lstinline!x2!, \lstinline!der(x1)!, \lstinline!der(x2)!, and \lstinline!y!.
+Two additional initial equations thus need to be provided to obtain a globally balanced initialization problem.
+Additionally those two initial equations must be chosen with care to ensure that they in combination with the dynamic equations give a well-determined initialization problem.
```

Comment:
Wouldn't this be even more readable?
```suggestion
Additionally, those two initial equations must be chosen with care to ensure that they give a well-determined initialization problem in combination with the dynamic equations.
```

Reply:
I prefer the first of those two variants; even if it takes longer to get to the "punch-line", since it more clearly shows that initial and dynamic equations are combined on an (almost) equal footing when making the initialization problem.